app/controllers/pages_controller.rb

```diff
@@ -1,7 +1,30 @@
 # frozen_string_literal: true
 
 class PagesController < ApplicationController
+  include ActiveModel::Validations
```

Comment:
Don’t think we need to do the validation in the controller so we? We can rely on the model to do the validation and generate the errors.

Reply:
Removed from controller